R/gt.R

```diff
@@ -83,17 +87,30 @@ gt <- function(data,
                id = random_id(),
                row_group.sep = getOption("gt.row_group.sep", " - ")) {
 
+  if (inherits(data, "grouped_df")) {
```

Comment:
What happens if a data frame is grouped, and also has `groupname_col`? What I'm seeing is that the `grouped_df` aspect wins.

```
mtcars %>% group_by(vs) %>% gt(groupname_col = "cyl")
```

Reply:
You could solve this by having `groupname_col` default to `dplyr::group_vars(data)`, perhaps?